TeensyDmx.cpp

```diff
@@ -1704,6 +2211,29 @@ void TeensyDmx::loop()
     if (m_rdmNeedsProcessing)
     {
         m_rdmNeedsProcessing = false;
-        processRDM();
+        if (m_mode == DMX_OUT) {
```

Comment:
Would this code better reside in processRDM rather than cluttering this function?

Reply:
Probably in a separate function perhaps; processRDM is all about being an RDM responder, this code is all about handling data related to being an RDM controller.

Reply:
A separate function sounds prudent.

Reply:
Done